test/test_ssl.py

```diff
@@ -400,20 +400,32 @@ def test_servers_no_disconnect_on_reload_with_no_tls_change(bouncer, pg, cert_di
 
 def test_servers_disconnect_when_changing_tls_config(bouncer, pg, cert_dir):
     bouncer.default_db = "pTxnPool"
-    bouncer.write_ini(f"server_tls_protocols = tlsv1.0")
+    bouncer.write_ini("server_lifetime = 2147483647")
+    bouncer.write_ini("server_idle_timeout = 2147483647")
+    bouncer.write_ini("server_tls_protocols = tlsv1.0")
     bouncer.admin("RELOAD")
 
     with bouncer.cur() as cur:
         assert pg.connection_count(dbname="p0") == 1
-        bouncer.write_ini(f"server_tls_protocols = secure")
+        bouncer.write_ini("server_tls_protocols = secure")
 
         with bouncer.log_contains(
             r"pTxnPool.*database configuration changed|pTxnPool.*obsolete connection", 1
         ):
             bouncer.admin("RELOAD")
-            for _ in wait_until("Did not close connection"):
-                if pg.connection_count(dbname="p0") == 0:
+
+            for _ in wait_until(
+                error_message="Connection did not closed",
+                timeout=5,
+                interval=0.5,
```

Comment:
Why change the default here?
```suggestion
                interval=0.5,
```

Reply:
Because if some one will change these default values, this test continue using wait_utils with "stable/tested" configuration.

If you want to provide a centralization management these options, let's define a special file with the such constants.